src/components/Select2.js

```diff
@@ -60,8 +60,10 @@ export default class Select2 extends Component {
           if (typeof item === 'string' ||
             ((!!item && typeof item === 'object') && Object.prototype.toString.call(item) === '[object String]')) {
             return (<option key={'option-' + k} value={item}>{item}</option>);
+          } else {
```

Comment:
Please, fix this: http://jmp.sh/L4JwgeQ ([rule](http://eslint.org/docs/rules/no-else-return.html))
The repository has a config for [eslint](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc)